sound/soc/intel/boards/sof_sdw.c

```diff
@@ -187,6 +187,16 @@ static const struct dmi_system_id sof_sdw_quirk_table[] = {
 					SOF_RT715_DAI_ID_FIX |
 					SOF_SDW_FOUR_SPK),
 	},
+	{
+		.callback = sof_sdw_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_SYS_VENDOR, "Intel"),
+			DMI_MATCH(DMI_PRODUCT_NAME, "adlrvp"),
```

Comment:
merged too quickly @kv2019i: this doesn't work for a UEFI BIOS.

Reply:
Oops, sorry @plbossart , I read the comments in wrong order... the comment on this was more recent than your approval. Let's do a fixup for this before sending upstream. Should we have another match case for UEFI case? @sathyap-chrome and @libinyang  ?

Reply:
@plbossart  
I requested coreboot to fix the match to be same as UEFI BIOS. once the fix is merged, i can send the fixup. maybe a day or 2.

Reply:
no worries, let's do a fixup

Reply:
@kv2019i , We need the quirk for UEFI. The DMI info is like what @fredoh9 provided.

Reply:
@plbossart  will share the new patch today itself